NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt

```diff
@@ -83,6 +90,8 @@ class NeuroID
         internal var locationService: LocationService? = null
 
         init {
+            setRemoteConfig()
```

Comment:
I am not a kotlin expert, because this function is marked as run blocking, you don't need an await or anything when you call it do you?

Reply:
await is in the remote config call. you can only call await in a suspend function. 

The runBlocking provides the coroutine that we can call the await in.  We then run the network call on a worker thread and suspend the main thread while the worker thread is doing stuff to avoid a warning from the OS to not run long running operations on the Main thread in the logs. Once the worker thread is done, the await() will release the blocking thread to run again. The output will look like this (logging added which is not in the PR): 

```
2024-03-29 10:50:57.836 16275-16275 System.out              com.sample.neuroid.us                I  remote start
2024-03-29 10:50:58.427 16275-16310 ****** Neu...***** init com.sample.neuroid.us                D  remoteConfig: NIDRemoteConfig(callInProgress=true, eventQueueFlushInterval=5, eventQueueFlushSize=2000, geoLocation=true, gyroAccelCadence=false, gyroAccelCadenceTime=200, requestTimeout=10)
2024-03-29 10:50:58.428 16275-16275 System.out              com.sample.neuroid.us                I  remote end
2024-03-29 10:50:58.428 16275-16275 System.out              com.sample.neuroid.us                I  remote store mgr start
2024-03-29 10:50:58.540 16275-16275 System.out              com.sample.neuroid.us                I  remote store mgr end
```

Reply:
Sweet, as long as we have mitigated that race condition I am good